src/main.cpp

```diff
@@ -535,10 +535,20 @@ With no FILE, start recording the current screen.
   -o, --output              Specify the output where the video is to be recorded.
 
   -p, --codec-param         Change the codec parameters.
-                            -p <option_name>=<option_value>
+                            -p <option_name>=<option_value>)");
+#ifdef HAVE_OPENCL
+    printf(R"(
+
+  -e, --opencl              Use the -e[#] or --opencl[=#] in conjunction with -t or --force-yuv option
+                            to use opencl for gpu accelerated conversion of data to yuv. # is one
+                            of the devices listed when running without specifying #.)");
```

Comment:
Have you tested this? I think you would need a newline at the end of this.

Reply:
It was working when I tested it, yes.